pyhive/presto.py

```diff
@@ -7,6 +7,9 @@
 
 from __future__ import absolute_import
 from __future__ import unicode_literals
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
I think this isn't kosher for library code, since this call mutates the standard library for the whole python process.

It's probably best to do it manually with
```
try:
    import urllib.parse as urlparse
except ImportError:
    # python 2
    import urlparse
```